docs/topics/dokkatoo-migration.md

```diff
@@ -0,0 +1,359 @@
+[//]: # (title: Migrate to Dokka Gradle plugin v2)
+
+> The Dokka Gradle plugin v2 is an [experimental](https://kotlinlang.org/docs/components-stability.html#stability-levels-explained) feature. 
+> It may be changed at any time. We appreciate your feedback in [GitHub](https://github.com/Kotlin/dokka/issues).
+>
+{type="warning"}
+
+The Dokka Gradle plugin (DGP) is a tool to generate comprehensive API documentation for Kotlin projects.
+
+DGP seamlessly processes both Kotlin's KDoc comments and Java's Javadoc comments to extract information and create 
+structured documentation in [HTML or Javadoc](#select-documentation-output-format) format.
+
+From Dokka 2.0.0, you can try the Dokka Gradle plugin v2, the new version of DGP.
+
+DGP v2 introduces significant improvements to DGP, aligning more closely with Gradle best practices. Key improvements are:
+
+* Adoption of Gradle types, leading to better performance.
+* Use of intuitive top-level DSL configuration instead of low-level task-based setups, simplifying the build scripts and their readability.
+* More declarative approach to documentation aggregation, making multi-project documentation easier to manage.
+* Plugin configuration is now typesafe, improving the reliability and maintainability of your build scripts.
+
+## Before you start
+
+Before diving into your project migration, complete the following preparatory steps according to your project.
+
+### Verify supported versions
+
+Ensure your project meets the minimum version requirements:
+
+| **Tool**              | **Version**   |
+|-----------------------|---------------|
+| Gradle                | 7.6 or higher |
+| Android Gradle plugin | 7.0 or higher |
+| Kotlin Gradle plugin  | 1.9 or higher |
+
+### Enable the new Dokka Gradle plugin
+
+1. Update the Dokka version to 2.0.0 in the `plugins {}` block of your project’s `build.gradle.kts` file:
+
+   ```kotlin
+   plugins {
+       kotlin("jvm") version "1.9.25"
+       id("org.jetbrains.dokka") version "2.0.0"
+   }
+   ```
+
+   > In DGP v2, this plugin ID outputs HTML by default. For more information about getting Javadoc output, see 
+   > [Select documentation output format](#select-documentation-output-format).
+   >
+   {type="tip"}
+
+2. In the project's `gradle.properties` file, set the following opt-in flag with helpers to activate the new plugin version:
+
+   ```kotlin
+   org.jetbrains.dokka.experimental.gradle.pluginMode=V2EnabledWithHelpers
+   ```
+
+   If your project does not have a `gradle.properties` file, create one in the root directory of your project.
+
+3. In the Gradle tool window, click the **Reload** button to sync the project in IntelliJ IDEA. This ensures the new 
```

Comment:
This assumes people are using IJ IDEA. It would be great to add Android Studio instructions as well. If not I would still reorganize it to be less assuming:
> If you are using IntelliJ IDEA, ...

Reply:
Got it. Added Android Studio. Please check.